quil-py/quil/program/__init__.pyi

```diff
@@ -120,6 +121,24 @@ class Program:
         Convert the instruction to a Quil string. If any part of the instruction can't
         be converted to valid Quil, it will be printed in a human-readable debug format.
         """
+    def wrap_in_loop(self, loop_count_reference: MemoryReference, iterations: int) -> "Program":
+        """
+        Return a copy of the `Program` wrapped in a loop that repeats ``iterations`` times.
+
+        The loop is constructed by wrapping the body of the program in classical Quil instructions.
+        The given ``loop_count_reference`` must refer to an INTEGER memory region. The value at the
+        reference given will be set to `iterations` and decremented in the loop. The loop will
```

Comment:
Should this be double-backtick'd?

```suggestion
        reference given will be set to ``iterations`` and decremented in the loop. The loop will
```

Reply:
Yeah, `pdoc` uses a markdown extension that makes it such that paths wrapped in single backticks links to that path in the documentation. Double backticks is normal inline code.

Not sure why the action hasn't updated the deployment notification on this PR, but [you can see how it renders in the preview here](https://rigetti.github.io/quil-rs/pr-preview/pr-321/quil/program.html#Program.wrap_in_loop).